index.html

```diff
@@ -237,7 +238,7 @@ <h3>
 
               <p>Note: after Level 2 of Navigation Timing is ratified, future development of new Navigation Timing features will be done within Resource Timing; Navigation Timing will be merged with Resource Timing.</p>
               <p>Latest publication: <a href="https://www.w3.org/TR/2017/WD-navigation-timing-2-20171205/">05 December 2017</a><br>Reference Draft:
-<a href="http://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/">http://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/</a><br>
+<a href="https://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/">https://www.w3.org/TR/2015/WD-navigation-timing-2-20150120/</a><br>
```

Comment:
@plehegar @siusin how did we derive these refs?

We have a newer publication @ https://www.w3.org/TR/2017/WD-navigation-timing-2-20171205/

Ditto for most of the other specs below.

Reply:
How about using the refs to the latest published version? i.e., https://www.w3.org/TR/navigation-timing-2/

Reply:
The "Reference Draft" is defined by Process, https://www.w3.org/2018/Process-20180201/#WGCharter for Patent Policy purposes. It's distinct from the "Latest Publication" draft from which the group will continue its work.